src/ray/common/bundle_spec.cc

```diff
@@ -36,30 +36,9 @@ void BundleSpecification::ComputeResources() {
 void BundleSpecification::ComputeBundleResourceLabels() {
   RAY_CHECK(unit_resource_);
 
-  for (size_t i = 0; i < unit_resource_->predefined_resources.size(); ++i) {
-    auto resource_name = scheduling::ResourceID(i).Binary();
-    const auto &resource_value = unit_resource_->predefined_resources[i];
-    if (resource_value <= 0.) {
-      continue;
-    }
-
-    /// With bundle index (e.g., CPU_group_i_zzz).
-    const std::string &resource_label =
-        FormatPlacementGroupResource(resource_name, PlacementGroupId(), Index());
-    bundle_resource_labels_[resource_label] = resource_value.Double();
-
-    /// Without bundle index (e.g., CPU_group_zzz).
-    const std::string &wildcard_label =
-        FormatPlacementGroupResource(resource_name, PlacementGroupId(), -1);
-    bundle_resource_labels_[wildcard_label] = resource_value.Double();
-  }
-
-  for (const auto &resource_pair : unit_resource_->custom_resources) {
-    auto resource_name = scheduling::ResourceID(resource_pair.first).Binary();
-    const auto &resource_value = resource_pair.second;
-    if (resource_value <= 0.) {
-      continue;
-    }
+  for (auto resource_id : unit_resource_->ResourceIds()) {
```

Comment:
`auto &resoure_id` ?

Reply:
I didn't use references because the resource id is just an integer. 
Although this is not likely to change, I can still add references as it doesn't hurt any way.